datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxUpgradeController.java

```diff
@@ -55,7 +56,7 @@ public SdxUpgradeResponse upgradeClusterByCrn(@TenantAwareParam @ResourceCrn Str
 
     @Override
     @InternalOnly
-    public SdxCcmUpgradeResponse upgradeCcm(@TenantAwareParam String environmentCrn) {
+    public SdxCcmUpgradeResponse upgradeCcm(String environmentCrn, @InitiatorUserCrn String initiatorUserCrn) {
```

Comment:
I think `@ValidCrn` and `@NotEmpty` annotations shall be also repeated here.

Reply:
I see no purpose for that

Reply:
I do not either, but they are validation annotations, so they must appear at both endpoint and controller level. :smile: At least last time I tried the request failed when the annotation was present in either but missing in the other one.